packages/ui/src/primitives/mui/IconButton/index.tsx

```diff
@@ -66,27 +67,29 @@ const IconButton = ({
   }
 
   return (
-    <MUIIconButton
-      id={id}
-      className={`${baseStyle} ${background ? styles.backgroundStyle : ''} ${className ?? ''}`}
-      disabled={disabled}
-      disableRipple={disableRipple}
-      size={size}
-      edge={edge}
-      sx={{
-        height: sizePx ? `${sizePx}px` : 'auto',
-        width: sizePx ? `${sizePx}px` : 'auto',
-        background: background ? `${background} !important` : 'auto',
-        ...sx
-      }}
-      title={title}
-      onClick={onClick}
-      onPointerUp={handleSoundEffect}
-      onPointerEnter={handleSoundEffect}
-      {...props}
-    >
-      {icon}
-    </MUIIconButton>
+    <Tooltip id={id} title={title}>
```

Comment:
What is the behaviour of icon buttons that are not given a title? We should probably be putting this outside of the IconButton component as needed, rather than here

Reply:
Right.
For this purpose, added a new element `IconButtonWithTooltip`